include/revng/ADT/STLExtras.h

```diff
@@ -37,6 +37,37 @@ static_assert(is_specialization_v<std::vector<int>, std::vector>);
 static_assert(is_specialization_v<const std::vector<int>, std::vector>);
 static_assert(is_specialization_v<std::pair<int, long>, std::pair>);
 
+//
+// always_true and always_false
+//
```

Comment:
Unused?

Reply:
For example, here: https://github.com/Cvelth/revng/blob/feature/generic_register_function_arguments/include/revng/Model/Register.h#L534

Reply:
Sure, sorry I didn't do so in the first place.